model/vcs/metrics.yaml

```diff
@@ -0,0 +1,139 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
```

Comment:
```suggestion
    brief: 'The number of changes (pull requests/merge requests) in a repository, categorized by their state (either open or merged)'
```

Reply:
Done in e174ef3572be947aa46d638168c8c287e6e32536.